package.json

```
@@ -28,7 +29,7 @@
         "react": "19.1.0",
         "react-dom": "19.1.0",
         "react-i18next": "15.5.1",
-        "react-router-dom": "7.5.2",
+        "react-router-dom": "^7.6.0",
```

```suggestion
        "react-router-dom": "7.6.0",
```

We prefer to lock dependencies in this codebase